proto/greenfield/virtualgroup/types.proto

```diff
@@ -35,10 +35,12 @@ message GlobalVirtualGroup {
 message GlobalVirtualGroupFamily {
   // id is the identifier of the global virtual group family.
   uint32 id = 1;
+  // primary_sp_id
+  uint32 primary_sp_id = 2;
```

Comment:
primary_sp_id should also be added in related events

Reply:
added